static/js/containers/FinancialAidCalculator.js

```diff
@@ -109,6 +124,16 @@ const validationMessage = (key, errors) => {
   </div>;
 };
 
+const apiError = ({ message, code }: FetchError) => (
+  <div className="api-error">
```

Comment:
Hmm. This isn't something that the user can take action on to resolve, but it's still something that they should be made aware of. This API error will prevent them from successfully submitting the form, right? If that's the case, maybe we should set `aria-invalid` and `aria-errormessage` on the `<form>` element, to indicate that the form can't be submitted but it's not due to any particular field being invalid.

I'll be honest, I don't actually know the best way to handle this situation. Maybe it's not worth handling right now. But the basic idea is, if this error message is important enough to show it to a sighted user, then it should also be important enough to be presented to non-sighted users as well.